sagemaker-python-sdk/mxnet_gluon_sentiment/sentiment.py

```diff
@@ -252,7 +252,7 @@ def get_dataset(filename):
     with open(filename) as f:
         for line in f:
             tokens = line.split()
-            label = int(tokens[0])
+            label = int(tokens[0].replace(':', '').replace(',', '').strip())
```

Comment:
Hi I was wondering if you have an example data that contains ':' here. I can understand we remove ',' so that '1,000' becomes '1000'. But I am worried for data like '100:2' which should be 50 and now becomes '1002'. 

I may be wrong. Just an example for ':' would be helpful to see what we should do.

Reply:
I don't have an example data, the error mentioned in PR description is from training job logs.

Regarding your concern, in this case if you pass '100:2' to int() it would fail too with ValueError.

Reply:
@neelamgehlot 

As we talked offline last time. Could you add a comment on this line indicating that you are doing data cleaning. Then we can go ahead to merge the codes.

Reply:
Added a comment in new commit